.githooks/pre-commit

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
have you made this file executable with `chmod +x` as well? I think git hooks by default aren't

Reply:
yeap

(.venv) ➜  radar-sdk-ios git:(pingxia/ch1646/add-clang-format-to-ios-sdk) ls -l .githooks/*
-rwxr-xr-x  1 pingxia  staff  742 Mar  1 22:33 .githooks/pre-commit